docs/events/glides/com.mbta.ctd.glides.trips_updated.v1.mdx

```diff
@@ -0,0 +1,288 @@
+
+An empty object `{}` is valid if nothing about the car has been modified.
+
+The length of `cars` always reflects the known length of the train.
+
+### DroppedReason
+
+An object representing the reason that a trip was dropped. The presence of this object implies that the trip was dropped.
+
+Fields:
+
+- `reason` (string): free-text description about why a trip was dropped.
+
+Currently the reason is entered by inspectors in a free-text field. It's published in an object so that if Glides collects more structured data in the future, then extra fields can be added. If more fields are added, then a generated text description would be filled into the `reason` field, and it would not be a breaking change.
+
+`reason` MAY be the empty string if for some reason Glides does not have information about why a trip was dropped.
+
+### Location
+
+See [Location](.#location)
+
+### Metadata
+
+See [Metadata](.#metadata)
+
+### Operator
+
+See [Operator](.#operator)
+
+### Scheduled
+
+Scheduled information about the trip. It was not updated in Glides, but is included in the event stream so that consumers can know the schedule information that Glides uses. If data here was never overriden by a corresponding field in [TripUpdated](#tripupdated), then consumers can assume that the trip operated based on the scheduled information contained here.
+
+Fields:
+
+- `scheduledCars` (array of [ScheduledCar](#scheduledcar)): Array of length 1 or 2. The length reflects the scheduled length of the train. In a 2 car train, the front car is listed first.
+
+It does not include time and location fields, because for scheduled trips those fields are already in [TripKey](#tripKey)
+
+### ScheduledCar
+
+Fields:
+
+- `run` (string, optional): The run number scheduled to the trip.
+- `operator` ([Operator](#operator), optional): The operator who is scheduled to operate the run that day, and therefore scheduled to do the trip.
+
+If fields are missing from ScheduledCar, it is not known who is scheduled to operate that car. An empty object is valid if we know the car is scheduled to operate but don't know anything else about it.
+
+### Time
+
+Time in the `HH:MM:SS` format. The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in `HH:MM:SS` local time for the day on which the trip schedule begins. Effectively, a [GTFS Time](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#field-types) but without support for `H:MM:SS` times.
+
+> _Example: `04:30:00` for 4:30AM or `25:35:00` for 1:35AM on the next day._
+
+Alternative: times as RFC3999 string.
```

Comment:
question: is this useful context for the schema documentation?

Reply:
A little? I can imagine someone reading this and thinking "why are they using this weird format instead of just using normal timestamps?". This is there to reduce the confusion of that hypothetical reader.

(Though in this specific case, we might end up switching to timestamps because they're better for representing unscheduled overnight nonrevenue trains. TBD.)

Reply:
If you're going to keep it, can you make it clear that this was an alternative considered for the schema, and not an alternative that's a valid value for this field?

Reply:
I've removed it.